package.json

```diff
@@ -17,9 +17,15 @@
   "license": "ISC",
   "description": "",
   "devDependencies": {
-    "babel": "^6.23.0",
-    "babel-cli": "^6.26.0",
-    "typescript": "^5.8.2"
-  },
-  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
+    "@scure/base": "^1.2.4",
+    "karma": "^6.4.4",
```

Comment:
Let's remove karma for now if it is not used,

Also, please add test command in the package, and integrate in the CI